plugins/AIAgents/lang/en.json

```diff
@@ -0,0 +1,37 @@
+{
+  "AIAgents": {
+    "AIAgentName": "AI Agent Name",
+    "AIAgentVisits": "AI Agent Visits",
+    "AIAgentVisitsDocumentation": "This is an overview of AI Agent visits.",
+    "AIAssistants": "AI Assistants",
```

Comment:
Wondering if we should use `Referrers_AIAssistants` instead of adding a new one here.